docs/data-sources/rest_managed.md

```diff
@@ -33,7 +33,8 @@ data "aci_rest_managed" "example" {
 * `class_name` - (string) Which class object is being created.
 * `id` - (string) The Distinguished Name of the object.
 * `annotation` - (string) Annotation for the class object that is being created. When annotation is provided in content this will take precedence.
-* `content` (map) Map of key-value pairs those needed to be passed to the Model object as parameters. Make sure the key name matches the name with the object parameter in ACI.
+* `content` - (map) Map of key-value pairs those needed to be passed to the Model object as parameters. Make sure the key name matches the name with the object parameter in ACI.
+* `escape_html` - (Boolean) Enable escaping of HTML characters when encoding the JSON payload.
```

Comment:
Do we need to expose this for the datasource, since we are only doing a get request to dn if not mistaken?

Reply:
Removed from datasource